core/src/main/resources/SpringApplicationContext.xml

```diff
@@ -262,7 +262,7 @@
 		name="messageListenerClassName"
 		class="java.lang.String"
 		>
-		<constructor-arg value="${jms.messageListenerClassName}"/>
+		<constructor-arg value="nl.nn.adapterframework.jms.IbisMessageListenerContainer"/>
```

Comment:
If this property should be removed then I think we should also remove this bean, which is just a string to configure that classname.
If the classname is now static then we don't need a bean for it.

However let's first decide if we can drop that property, yay or nay.

Reply:
I agree. I cleaned this part as well.